x-pack/legacy/plugins/apm/public/components/app/Main/Home.tsx

```diff
@@ -43,6 +44,8 @@ const SETTINGS_LINK_LABEL = i18n.translate('xpack.apm.settingsLinkLabel', {
 });
 
 export function Home() {
+  useTrackVisit({ app: 'apm', path: 'home' });
+  useTrackVisit({ app: 'apm', path: 'home', delay: 15000 });
```

Comment:
This doesn't really work as-is -- it logs this "home" visit on both the services and traces tabs. Also none of the APM visits wait to see if the user has any data (although I'm not 100% sure I agree with Rasmus that was important, if they're using the UI?). If that's necessary, we can easily supplement this strategy with other event tracking to capture that kind of thing. I'm curious to see if we merge something like this, how this data compares to the existing data.

Reply:
Currently we track

Reply:
(Simple fix would be to remove these calls and use the hook at the services and traces component level to separate those views)

Reply:
This is fixed: moved these to services and traces